frontend/src/components/skeletons/Card.tsx

```
@@ -31,6 +31,7 @@ const CardSkeleton: React.FC<CardSkeletonProps> = ({
 
           {showIcons && (
             <div className="flex min-w-[30%] grow flex-row items-center justify-start gap-2 overflow-auto">
+              // NOSONAR As safe to use index as key - static skeleton items with fixed length
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)